tests/unit/editorconfig-to-prettier.js

```diff
@@ -63,9 +63,7 @@ test("editorconfigToPrettier", () => {
       tab_width: 0,
       indent_size: 0,
     }),
-  ).toStrictEqual({
-    tabWidth: 0,
-  });
```

Comment:
The value `0` is invalid, discussed in https://github.com/josephfrazier/editorconfig-to-prettier/issues/1 and decided to keep it, but [`tabWidth=0` crashes](https://github.com/prettier/prettier/issues/7388), I think better to ignore instead.